src/main/java/org/opentripplanner/middleware/tripMonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -332,19 +440,39 @@ public boolean shouldSkipMonitoredTripCheck() throws Exception {
         // get the configured timezone that OTP is using to parse dates and times
         ZoneId targetZoneId = DateTimeUtils.getOtpZoneId();
 
+        // Make sure journey state indicates that the trip should still possible on at least one day of the week as of
+        // the last time a matching itinerary was attempted to be found. This field in the journey state is used instead
+        // of the trip's itinerary existence data in order to not skip trips that are monitored only one day of the week
+        // and may have not had a matching itinerary on that single monitored day even though it could be possible the
+        // next week.
```

Comment:
Might be simpler to say:

```
        // If trip is no longer possible, no further checking is needed. We should not check itinerary existence data
        // here to avoid incorrectly skipping trips that are monitored on a single day of the week, but which may have
        // not had a matching itinerary on that day for one week (even though the trip is possible the next week).
```

Reply:
Updated in 1987213b6754ffa98f4748820b628be64f9443f4